extensions/markdown-language-features/src/features/previewContentProvider.ts

```diff
@@ -84,7 +86,7 @@ export class MarkdownContentProvider {
 					data-state="${escapeAttribute(JSON.stringify(state || {}))}">
 				<script src="${this.extensionResourcePath('pre.js')}" nonce="${nonce}"></script>
 				${this.getStyles(sourceUri, nonce, config, state)}
-				<base href="${markdownDocument.uri.with({ scheme: 'vscode-resource' }).toString(true)}">
+				<base href="${toResoruceUri(markdownDocument.uri)}">
```

Comment:
The problem discussed here https://github.com/microsoft/vscode/pull/74332#issuecomment-504677588 is because of the change in this line.
```ts
// before, <base>'s href were a string
markdownDocument.uri.with({ scheme: 'vscode-resource' }).toString(true) === "vscode-resource://desktop-9su47m5/test-vscode-issue/test.md"
// here, it has two slashes: ----------------------------------------------------------------^^

// after, <base>'s href is an object
toResoruceUri(markdownDocument.uri) === { $mid: 1, path: "/test-vscode-issue/test.md", scheme: "vscode-resource" }
toResoruceUri(markdownDocument.uri).toString() === "vscode-resource:/test-vscode-issue/test.md"
// the missing stash is coming from here: --------------------------^
```

Reply:
It isn't just the missing slash, it is loosing the URI's authority too ("desktop-9su47m5" in this example).